bouter/embedded/__init__.py

```diff
@@ -1,25 +1,45 @@
-import utilities
-from bouter import Experiment
-import flammkuchen as fl
 import numpy as np
+import pandas as pd
+
+from bouter import utilities, decorators, bout_stats
+from bouter import Experiment
 
 
 class EmbeddedExperiment(Experiment):
-    def __init__(self, *args, **kwargs):
+    def __init__(self, *args, continue_curvature=5, **kwargs):
         super().__init__(*args, **kwargs)
 
-    def vigor(self, vigor_duration=0.05):
+        if continue_curvature:
```

Comment:
fill_out_segments also works for the continue_curvature being 0

Reply:
Ok, then let's have the check with if None to skip